packages/@roots/bud-dashboard/src/hooks/useCompilation.ts

```diff
@@ -62,14 +58,13 @@ export const useCompilation: Dashboard.Compilation.Hook = app => {
 
     app.compiler.compile(app.build.config)
 
-    app.isProduction &&
-      app.compiler.instance.hooks.done.tap(app.name, () =>
-        app.compiler.instance.close(err => {
-          err && setErrors([err])
-          setClosed(true)
-          setTimeout(process.exit, 2000)
-        }),
-      )
+    app.compiler.instance.hooks.done.tap(app.name, () =>
```

Comment:
This is kind of unrelated tbh, but an important fix for dev build caching.

We want the `close` fn to be called by dev (so that cache is consistently written).. but we don't want the process to be closed. The way it was before skipped the whole call, which isn't what we want.